src/components/color-picker/color-picker.tsx

```diff
@@ -102,7 +102,11 @@ export class ColorPicker implements FormComponent {
         }
 
         return (
-            <limel-popover open={this.isOpen} onClose={this.onPopoverClose}>
+            <limel-popover
+                open={this.isOpen}
+                openDirection="bottom-start"
```

Comment:
Since this can't be set by the consumer, and there is no change to the what the user sees, it's "part of" the main change to limel-portal. So I've squashed this commit into that one.